src/input.cpp

```diff
@@ -0,0 +1,396 @@
+};
+
+const char *steam_input_types[] = {
+    "Unknown", "SteamController", "XBox360Controller", "XBoxOneController", "GenericXInput", "PS4Controller", "AppleMFiController", "AndroidController", "SwitchJoyConPair", "SwitchJoyConSingle", "SwitchProController", "MobileTouch", "PS3Controller", nullptr,
+};
+
+const char *xbox_origins[] = {
+    "A", "B", "X", "Y", "LeftBumper", "RightBumper", "Menu", "View", "LeftTrigger_Pull", "LeftTrigger_Click", "RightTrigger_Pull", "RightTrigger_Click", "LeftStick_Move", "LeftStick_Click", "LeftStick_DPadNorth", "LeftStick_DPadSouth", "LeftStick_DPadWest", "LeftStick_DPadEast", "RightStick_Move", "RightStick_Click", "RightStick_DPadNorth", "RightStick_DPadSouth", "RightStick_DPadWest", "RightStick_DPadEast", "DPad_North", "DPad_South", "DPad_West", "DPad_East", "Count", nullptr,
+};
+
+}
+
+// void ActivateActionSet( InputHandle_t inputHandle, InputActionSetHandle_t actionSetHandle );
+EXTERN int luasteam_activateActionSet(lua_State *L) {
+    uint64 inputHandle;
+    if (lua_isstring(L, 1)) {
+        const char *controller = luaL_checkstring(L, 1);
+        if (controller == "all")
```

Comment:
Nice solution to use a string `all` here. But if it's not all, something should happen. You probably shouldn't call the function, and maybe print some error? Failing might also be ok, not sure if too harsh.